packages/discovery-react-components/src/components/SearchFacets/utils/searchFilterTransform.tsx

```diff
@@ -8,10 +8,23 @@ import {
   SelectableDynamicFacets
 } from './searchFacetInterfaces';
 
+/*
+ * Utility class for creating and parsing query strings for search filters
+ */
+
 export class SearchFilterTransform {
-  static SPLIT_UNQUOTED_COMMAS = /,(?=(?:(?:[^"\\"]*["\\"]){2})*[^"\\"]*$)/;
-  static SPLIT_UNQUOTED_COLONS = /:(?=(?:(?:[^"\\"]*["\\"]){2})*[^"\\"]*$)/;
-  static SPLIT_UNQUOTED_PIPES = /\|(?=(?:(?:[^"\\"]*["\\"]){2})*[^"\\"]*$)/;
+  // Match any unescaped commas, colons, pipes
+  static SPLIT_UNESCAPED_COMMAS = /(?<!\\),/;
+  static SPLIT_UNESCAPED_COLONS = /(?<!\\):/;
+  static SPLIT_UNESCAPED_PIPES = /(?<!\\)\|/;
+  // Match if the string is surrounded by parens
+  // The substring inside of the parens will be in $1
+  static STRING_IN_PARENS = /^\((.+)\)$/;
+  // Match any special characters
```

Comment:
Link to https://cloud.ibm.com/docs/discovery-data?topic=discovery-data-query-operators, if that's what you're using to determine "special characters".